packages/core/src/indexing/postgres/postgres-index-api.ts

```diff
@@ -0,0 +1,54 @@
+import { StreamID } from 'streamid/lib/stream-id.js'
+import type { BaseQuery, Pagination, Page } from '@ceramicnetwork/common'
+import type { DatabaseIndexApi, IndexStreamArgs } from '../database-index-api.js'
+import { initTables } from '../postgres/init-tables.js'
+import { InsertionOrder } from '../postgres/insertion-order.js'
+import { asTableName } from '../as-table-name.util.js'
+import { Knex } from 'knex'
+
+export class PostgresIndexApi implements DatabaseIndexApi {
+  private readonly insertionOrder: InsertionOrder
+  constructor(private readonly dbConnection: Knex, readonly modelsToIndex: Array<StreamID>) {
+    this.insertionOrder = new InsertionOrder(dbConnection)
+  }
+
+  public getActiveModelsToIndex(): Array<StreamID> {
+    /**
+     * Helper function to return array of active models that are currently being indexed by node
+     * as defined in the config file.
+     * TODO: extend to runtime check once adminAPI unlocks to add and load models on the fly
+     */
```

Comment:
When you do `TODO` like this, there has to be a corresponding issue on Linear.

`TODO(NET-XXX)` would then allow us to recover context and act on it.